### PR TITLE
Transform trix input to safe_html to prevent XSS vulerability.

### LINKED ISSUE #1529: Trix fields not XSS safe
We must add additional server side validation for trix fields to make them XSS safe.

### PR DESCRIPTION
**Pull request by deiferni**
This PR prevents vulerability to XSS by transforming trix input to safe_html using a plone transform on:

- The protocol page (where input fields are rendered manually)
- For the trix widget using a `DataConverter`

Closes #1529.

### CODE REVIEW

**Comment by deiferni** [comment]
@lukasgraf plz have a look

File: opengever/meeting/browser/meetings/protocol.py [hunk 1]
```diff
@@ -183,8 +183,17 @@ def unlock(self):
     def applyChanges(self, data):
         ModelEditForm.applyChanges(self, data)
 
+        transformer = api.portal.get_tool('portal_transforms')
```
Comment (lukasgraf) [inline opengever/meeting/browser/meetings/protocol.py]: As discussed: Move this down into the `update()` method.

**Comment by deiferni** [comment]
@lukasgraf updated (method moved)

**Comment by lukasgraf** [comment]
:+1:

**Pull request merged**